proto/common.proto

```diff
@@ -24,70 +24,40 @@ message SearchFilter {
   oneof has_max_road_class {
     RoadClass max_road_class = 2; // highest road class to allow in loki results [default = kMotorway]
   }
+
   // form of way
-  oneof has_exclude_tunnel {
-    bool exclude_tunnel = 3;  // whether to exclude tunnels from loki results [default = false]
-  }
-  oneof has_exclude_bridge {
-    bool exclude_bridge = 4;  // whether to exclude bridges from loki results [default = false]
-  }
-  oneof has_exclude_ramp {
-    bool exclude_ramp = 5;    // whether to exclude roads with ramp use from loki results [default = false]
-  }
+  bool exclude_tunnel = 3;  // whether to exclude tunnels from loki results [default = false]
+  bool exclude_bridge = 4;  // whether to exclude bridges from loki results [default = false]
+  bool exclude_ramp = 5;    // whether to exclude roads with ramp use from loki results [default = false]
   oneof has_exclude_closures {
     bool exclude_closures = 6;// whether to exclude roads marked as closed due to traffic [default = true]
   }
 }
 
 message PathEdge {
-  oneof has_graph_id {
-    uint64 graph_id = 1;
-  }
-  oneof has_percent_along {
-    double percent_along = 2;
-  }
+  uint64 graph_id = 1;
+  double percent_along = 2;
   LatLng ll = 3;
-  oneof has_side_of_street {
-    Location.SideOfStreet side_of_street = 4;
-  }
-  oneof has_distance {
-    double distance = 5;
-  }
-  oneof has_minimum_reachability {
-    int32 minimum_reachability = 6; //deprecated
-  }
-  oneof has_begin_node {
-    bool begin_node = 7;
-  }
-  oneof has_end_node {
-    bool end_node = 8;
-  }
+  Location.SideOfStreet side_of_street = 4;
+  double distance = 5;
+  bool begin_node = 7;
+  bool end_node = 8;
   repeated string names = 10;
-  oneof has_outbound_reach {
-    int32 outbound_reach = 11;
-  }
-  oneof has_inbound_reach {
-    int32 inbound_reach = 12;
-  }
-  oneof has_heading {
-    float heading = 13;
-  }
+  int32 outbound_reach = 11;
+  int32 inbound_reach = 12;
+  float heading = 13;
 }
 
 // Output information about how the location object below is correlated to the graph or to the route etc
 message Correlation {
-  repeated PathEdge edges = 50;
-  repeated PathEdge filtered_edges = 51;
-  oneof has_original_index {
-    uint32 original_index = 52;
-  }
-  LatLng projected_ll = 53;
-  oneof has_leg_shape_index {
-    uint32 leg_shape_index = 54;
-  }
-  oneof has_distance_from_leg_origin {
-    double distance_from_leg_origin = 55;
-  }
+  repeated PathEdge edges = 1;
```

Comment:
any  concerns renumbering the fields here and few other places? I remember we store the trip leg pbf for some of the pinpoint tests which would fail to read back the correct data if the pbf fields were renumbered.

I guess since the tests are passing the renumbering here does not affect the few places we've stored these protobufs on disk?

Reply:
oh it broke the tests, but i fixed them by leaving the old defs there and copying them to the new defs and overwiiting the files. ive been doing this alot for all the protobuf changes, its a huge pain in the butt but its worth it to get the code cleaner. you see how it did it over here (takes a long time to load because of diff size): https://github.com/valhalla/valhalla/pull/3527/files#diff-836ee7bf7651f8100e6b6072ab85387a7fee2ea9178762aabf200726bd31a0d9R32

Reply:
the other stuff i renumbered were things that were parsed from the request, so i could just call `ParseApi` and then again overwrite the proto after i've parsed the request. Also a pain in the butt but something ive been doing a lot with all these changes and organizational stuff

Reply:
ah ok I missed some of those diffs at first glance (as you said, because of the size of the change github had issues expanding all the diffs).

thanks for updating the tests! :+1: